docs/specs/iota_did_method_spec.md

```diff
@@ -0,0 +1,324 @@
+* Set the `previous_message_id` field to the IOTA MessageId value.
+* Create a `proof` object referencing the public key used inside the `verificationMethod` field and the signature suite in the `type` field. 
+* Create a cryptographic signature using the same keypair and add the result in the appropriate field(s) inside the `proof` object.
+
+### Delete
+
+In order to deactivate a DID document, an Integration DID message must be published that removes all content from a DID Document, effectively deactivating the DID Document. Keep in mind that this is irreversable. 
+
+## IOTA Identity standards
+
+The `did:iota` method is implemented in the IOTA Identity framework. This framework supports a number of operations that are standardized, some are standardized across the SSI community, and some are the invention of the IOTA Foundation. 
+
+### Standardized Verification Method Types
+
+We support two different Verification Method Types. Verification methods that can be used for signing DID Document, and verification methods that are only used for signing [W3C standardized Verifiable Credentials](https://www.w3.org/TR/vc-data-model/). This set differs as the IOTA Identity implements revocation through public key removal, which means every Verifiable Credential must be signed by a different keypair. We create collection of keypairs in a Merkle Tree in order to save space in the DID Document.
+
+Verification Methods that can be used to sign DID Documents and do other reusable activities:
+* `Ed25519VerificationKey2018` to create a `JcsEd25519Signature2020`.
+
+Verification Methods that can be used to sign Verifiable Credentials:
+* `MerkleKeyCollection2021` to create a `MerkleKeySignature2021`.
+
+### Revocation
+
+As mentioned above, revocation is done through deactivating public keys in the IOTA Identity framework. IOTA Identity has a highlky scalable solution since some identities might sign and want to revoke thousands if not millions of verifiable credentials. It is also GDPR compliant, leaving no trace, not even a hash, of a verifiable credential on the Tangle. Below will be a brief overview of how this mechanism works, but you may read all the details in the MerkleKeyCollection standardization document. 
+
+#### Key Collections
+
+Instead of storing individual public keys in a DID Document, IOTA Identity introduces the `MerkleKeyCollection2021` verification method. It supports a REQUIRED `publicKeyBase58` field that MUST contain the top hash of a Merkle Tree. In this Merkle Tree all the individual leaves are a public keys using the Signature Algorithm of choice and Digest Algorithm of choice for the Merkle Tree process. This process allows the creation of millions of public keys, within a single verification Method and without bloating the DID Document. For specific info such as the maximum depth of the Merkle Tree, supported signature algorithms and digest algorithms can be found in the specification document. 
```

Comment:
leaves are a public keys -> leaves are public keys
Signature Algorithm -> signature algorithm
Digest Algorithm -> digest algorithm
public keys, within a single -> public keys within a single  (no comma)
Method -> method
For specific info such -> Specific info, such